src/SprykerMiddleware/Service/Process/Model/JsonStreamService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SprykerMiddleware\Service\Process\Model;
+
+class JsonStreamService implements StreamServiceInterface
+{
+    /**
+     * @param resource $stream
+     *
+     * @return array
+     */
+    public function read($stream)
+    {
+        return json_decode(fgets($stream), true);
```

Comment:
Do we need this service to actually support streams? https://www.salsify.com/blog/engineering/json-streaming-parser-for-php